03-praticas/utils/utils.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import (
+	"errors"
+	"strconv"
+	"strings"
+	"unicode"
+)
+
+func StringToSliceInt(numString string, hasVirgula bool) ([]int, error) {
+	numArray := []int{}
+	var numStrs []string
+
+	if hasVirgula {
+		numStrs = strings.Split(strings.TrimSpace(numString), ",")
+	} else {
+		numStrs = strings.Split(strings.TrimSpace(numString), "")
+	}

```

Comment:
Si bien está bien, es mejor que sea un switch, dado que el formato para separar puede crecer y, nada mejor que comprender esto con un switch

```suggestion
	    switch {
    case hasVirgula:
        numStrs = strings.Split(strings.TrimSpace(numString), ",")
    default:
        numStrs = strings.Split(strings.TrimSpace(numString), "")
    }
```